templates/partials/committee-summary.html

```diff
@@ -20,14 +20,18 @@ <h4 class="figure__title">Totals for {{totals.cycle|fmt_year_range|default("unav
   <div class="table__half">
     <div class="table__row">
       <div class="table__cell"><span class="term" data-term="Cash on Hand">Cash on hand</span></div>
-      <div class="table__cell">{{reports.cash_on_hand_end_period|currency|default("unavailable")}}</div>
+      <div class="table__cell">{{report.cash_on_hand_end_period|currency|default("unavailable")}}</div>
     </div>
     <div class="table__row">
       <div class="table__cell"><span class="term" data-term="Debt">Debt</span></div>
-      <div class="table__cell">{{reports.debts_owed_by_committee|currency|default("unavailable")}}</div>
+      <div class="table__cell">{{report.debts_owed_by_committee|currency|default("unavailable")}}</div>
     </div>
     <div class="table__row--note">
-      Source: <a href="http://docquery.fec.gov/pdf/{{ reports.beginning_image_number|last_n_characters }}/{{ reports.beginning_image_number }}/{{ reports.beginning_image_number }}.pdf">{{reports.report_type_full|fmt_report_desc|default("unavailable")}} {{ reports.report_year }}</a>
+      {% if report %}
+      Source: <a href="{{ report|url_to_fec_pdf }}">{{report.report_type_full|fmt_report_desc|default("unavailable")}} {{ report.report_year }}</a>
```

Comment:
Well in this case, the "unavailable" is only in the case that we don't have a full report description. We _ought_ to have that if we have a report, but I'm not 100% certain if that will always be the case. So what if we replace the default link text "unavailable" with something like `default("Committee Report")`?